config/application.rb

```diff
@@ -37,5 +37,7 @@ class Application < Rails::Application
     config.after_initialize do
       Rails.error.subscribe(ErrorSubscriber.new)
     end
+
+    config.active_storage.service_urls_expire_in = 15.minutes
```

Comment:
I wonder if we should make it longer? Also, what do you think about implementing a more permanent solution that refreshes the URL?